3-extra/3-card-validator.js

```diff
@@ -0,0 +1,73 @@
+// ## **PROJECT: Credit Card Validator**
+
+// In this project you'll write a script that validates whether or not a credit card number is valid.
+
+// Here are the rules for a valid number:
+
+// - Number must be 16 digits, all of them must be numbers.
+// - You must have at least two different digits represented (all of the digits cannot be the same).
+// - The final digit must be even.
+// - The sum of all the digits must be greater than 16.
+
+// For example, the following credit card numbers are valid:
+
+// ```markdown
+// 9999777788880000
+// 6666666666661666
+// ```
+
+// And the following credit card numbers are invalid:
+
+// ```markdown
+// a92332119c011112 (invalid characters)
+// 4444444444444444 (only one type of number)
+// 1111111111111110 (sum less than 16)
+// 6666666666666661 (odd final number)
+// ```
+
+// These are the requirements your project needs to fulfill:
+
+// - Make a JavaScript file with a name that describes its contents.
+// - Create a function with a descriptive name which makes it clear what the function does. The function should take one argument, the credit card number to validate.
+// - Write at least 2 comments that explain to others what a line of code is meant to do.
+// - Return a boolean from the function to indicate whether the credit card number is valid.
+// - Use `node` from the command line to test if your code works as expected.
+
+// Good luck!
+
+function cardValidator (cardNumber){
+    let sixDigits = cardNumber.length == 16
```

Comment:
don't be afraid of using more characters if means being more precise. Because this "sixDigits" even if you mean as a short way of sixteenDigits can be misleading because it means a totally different thing.
Also ,the name sixDigits seem to suggest that the object returned will be something containing six digits, while what you are returning is a boolean. 
Some people like to store the boolean variable in a predictive way: isCarBlue hasChildren 
```
let digitLimit = 16
let hasValidNumberOfDigits =  cardNumber.length == digitLimit
```
but it is not set in stone and you could just go with
```
let digitLimit = 16
if (cardNumber.length !== 16) return false;

```
Since you want ALL the value to be true, as soon as you find a false value you can return the function and not analyze any of the following conditions

Reply:
changed names of variable, left untouched last check, but I totally agree to use on true or false from very beginning